src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5454,6 +5454,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_NullabilityMismatchInArgumentForOutput_Title" xml:space="preserve">
     <value>Argument cannot be used as an output for parameter due to differences in the nullability of reference types.</value>
   </data>
+  <data name="WRN_NullDisallowedInAssignment" xml:space="preserve">
+    <value>A possible null value may not be passed to a target marked with the [DisallowNull] attribute</value>
```

Comment:
Would it make sense to make the message more descriptive and include information for the parameter and the method as we do in other similar warnings?

Reply:
The reason I used general terms is because we'll use the same message for passing a maybe-null value to a property or field marked with `[DisallowNull]`.

---
In reply to: [292590685](https://github.com/dotnet/roslyn/pull/36263#discussion_r292590685) [](ancestors = 292590685)